src/js/media-container.js

```diff
@@ -306,6 +316,11 @@ class MediaContainer extends window.HTMLElement {
 
       // Stay visible if hovered over control bar
       this.removeAttribute('user-inactive');
```

Comment:
Very much unrelated, but noticed it when looking here. I'm not sure that it matters much in the real world, but thought I'd mention it here anyway.
We're adding and removing the `user-inactive` attribute unconditionally, particularly on mousemove. This causes devtools to think that the element has updated a lot. Particularly, as you move the mouse around on the various components of the control bar you can see devtools highlight `media-controller` showing that it's been updated, even if the update is removing a non-existent attribute.

Reply:
Good find

Reply:
Yeah, not ideal, but also should be prioritized appropriately, i.e. if it requires a big re-architecture we should consider if the issue is worth it right now. After the call this morning it sounds like @cjpillsbury thinks it may not be a big effort, or we could fix it per-attribute as needed.

Reply:
yeah, I don't think it's a *big* issue, but it's worth addressing. Theoretically, it's as easy as checking if the prop/attr has changed or not or only removing/add the attribute if the aren't added/removed as appropriate.